hypervisor/src/arch/mod.rs

```diff
@@ -1,3 +1,5 @@
+// Copyright © 2024 Institute of Software, CAS. All rights reserved.
+//
```

Comment:
The changes in this file do not look copyrightable to me.

Reply:
Noted, will remove in next force push :)